### DISCUSSION TITLE
INSPIRE Registry unavailablility due to maintenance

### DISCUSSION THREAD

**Discussion by fabiovinci** [INSPIRE-MIF/helpdesk-validator#991] [Announcements]
Dear Community,

due to the unavailability of the INSPIRE Registry service from **FRIDAY 11 August 12:00** until **WEDNESDAY 16 August 12:00** ([see related announcements](https://github.com/INSPIRE-MIF/helpdesk-registry/discussions/63)), you may encounter errors in the validations performed in the aforementioned period.

Sorry for the inconvenience.

**Comment by jenriquesoriano** [comment]
Dear Community,

the INSPIRE Registry service is back to operation according to the planned maintenance.

Best regards,
